app/src/main/java/org/sopt/havit/ui/save/SaveFragment.kt

```diff
@@ -47,11 +50,25 @@ class SaveFragment(categoryName: String) : BottomSheetDialogFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
         setBottomSheetShow()
+        setUrlPaste()
         setListeners()
     }
 
+    private fun setUrlPaste() {
+        clipboard = activity?.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
+        clipDate = clipboard.primaryClip!!
+
+        if (clipboard.hasPrimaryClip()) {
+            clipDate.apply {
+                val textToPaste: String = this.getItemAt(0).text.toString().trim()
+                binding.tvSaveUrl.text = textToPaste
+            }
+        } else {
+            binding.clSaveUrl.isVisible = false
+        }
```

Comment:
저의 취향이긴 하지만 
```suggestion
    } else { binding.clSaveUrl.isVisible = false  }
}
```
요론 느낌도 깔끔할듯 합니당 ㅎㅎ

Reply:
> 저의 취향이긴 하지만
> 
> 요론 느낌도 깔끔할듯 합니당 ㅎㅎ

ㅋㅋㅋㅋㅋㅋㅋ무슨 차이인지 한참 보고 알았습니다🤣🤣 참고할게요~~